chapters/negative_binomial_20250310.qmd

```diff
@@ -36,6 +38,152 @@ options(dplyr.summarise.inform = FALSE)
 
 
 
+## Background
+
+The **Negative Binomial distribution** extends the Binomial model to situations where the number of trials is not fixed in advance.  
+Instead, trials continue until a fixed number of **successes** \(r\) occurs, and the distribution describes the number of **failures** before the final success.  
+
+- **Connection to Binomial:** While the Binomial counts successes in a fixed number of trials, the Negative Binomial counts failures until reaching \(r\) successes.  
+- **Connection to Geometric:** When \(r=1\), the NB reduces to the Geometric distribution.  
+- **Key property:** It naturally accounts for **overdispersion**, i.e., variance greater than the mean, making it more flexible than the Poisson distribution for real-world count data.  
+
+The NB distribution is widely applied in **biostatistics, epidemiology, and public health** to model count outcomes such as number of hospital visits, infectious disease cases, or equipment failures, especially when variability exceeds Poisson expectations.
+
+
+### Modeling Count Data
```

Comment:
Can you move this to an "application" section at the end of the lesson? It's putting quite a bit of derivation using other distributions before we've even worked with the NB.